src/vs/code/browser/workbench/workbench-dev.html

```diff
@@ -25,6 +25,9 @@
 		<!-- Builtin Extensions (running out of sources) -->
 		<meta id="vscode-workbench-builtin-extensions" data-settings="{{WORKBENCH_BUILTIN_EXTENSIONS}}">
 
+		<!-- Product Overrides (running out of sources) -->
```

Comment:
I am curious why we do this actually only for running out of sources (also on desktop)?